local-volume/README.md

```diff
@@ -54,17 +74,46 @@ Future features:
 
 ### Step 1: Bringing up a cluster with local disks
 
+#### Setting the alpha feature gates
```

Comment:
/Setting the/Enable/

Plus add link anchor here to be used below, e.g. `<a name="feature-gates"></a>`.